src/Card/README.md

```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 </>
 ```
 
+### Card Content Block Empty
+
+With image
+
+```jsx live
+<Card style={{ width: '25rem' }}>
+  <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" />
+  <Card.Section className="text-center">
+    <h3>Headline</h3>
```

Comment:
Per the Figma spec, it looks like the heading is an `h2` in the examples. Should the example here reflect that? Similar question for the "Without image" example.

Reply:
Fixed to `h2`